packages/shr-fhir-export/lib/ig.js

```diff
@@ -1004,28 +1004,20 @@ ${match[1]}
 
   // Rewrite the updated CodeSystems HTML file
   const codeSystemsHtmlPath = path.join(outDir, 'pages', 'codesystems.html');
-  const codeSystemsHtml = fs.readFileSync(codeSystemsHtmlPath, 'utf8');
-  const codeSystemsDisclaimer = `<p>The code systems listed below include:</p>
-  <ol>
-    <li>code systems used in the definition of value sets in this IG; and</li>
-    <li>code systems for individual codes used directly in the logical models and profiles.</li>
-  </ol>
-  <p>This list is not inclusive of code systems associated with external value sets used in the IG.</p>`;
-  let updatedCodeSystemsHtml = codeSystemsHtml.replace('<code-systems-go-here/>', htmlCodeSystems.join(''))
-    .replace('<code-systems-disclaimer/>', codeSystemsDisclaimer);
-  if (hideSupporting) {
-    updatedCodeSystemsHtml = updatedCodeSystemsHtml
-      .replace('<code-systems-header/>', '<h2>Primary code systems used in this Implementation Guide</h2>');
-  } else {
+  if (htmlCodeSystems.length > 0) {
+    const codeSystemsHtml = fs.readFileSync(codeSystemsHtmlPath, 'utf8');
+    const codeSystemsDisclaimer = `<p>The code systems listed below are used by the
+    local value sets, logical models, and profiles shown on their respective pages in this IG.</p>
+    <p>This list is not inclusive of code systems associated with external value sets used in the IG.</p>`;
+    let updatedCodeSystemsHtml = codeSystemsHtml.replace('<code-systems-go-here/>', htmlCodeSystems.join(''))
+      .replace('<code-systems-disclaimer/>', codeSystemsDisclaimer);
     updatedCodeSystemsHtml = updatedCodeSystemsHtml
       .replace('<code-systems-header/>', '<h2>Code systems used in this Implementation Guide</h2>');
-  }
-  if (htmlCodeSystems.length === 0) {
-    updatedCodeSystemsHtml = updatedCodeSystemsHtml
-      .replace('<table class="codes">', 'None\n<table class="codes" style="display: none">');
-  }
 
-  fs.writeFileSync(codeSystemsHtmlPath, updatedCodeSystemsHtml, 'utf8');
+    fs.writeFileSync(codeSystemsHtmlPath, updatedCodeSystemsHtml, 'utf8');
+  } else {
+    fs.unlinkSync(codeSystemsHtmlPath);
```

Comment:
This line makes sense and seems like the best practice, but I think it is a little inconsistent with how we handle other tabs isn't it? For example, even if we don't have a conformance/capability statement, or a graph viewer in the navbar, it looks like we still leave the html pages laying around in `fhir/guide/pages`. If it isn't a difficult change, could you make it so those are removed too like the codesystems? If it seems like a difficult task, I'd just leave it and I will approve, because everything else looks fine.

Reply:
This is out of scope for this task, but the issue is logger here: https://standardhealthrecord.atlassian.net/browse/CIMPL-122.